app.js

```diff
@@ -19,16 +20,18 @@ logger.configure({
 });
 
 const apiRouter = express.Router();
-const api = require('./lib/routes/api')(apiRouter);
+// TODO: How to fix for eslint?
+const api = require('./lib/routes/api')(apiRouter); // eslint-disable-line no-unused-vars
```

Comment:
We handle this in Gambit by defining the Router within the route: https://github.com/DoSomething/gambit/blob/3.9.1/app/routes/index.js#L4 

Would that be cleaner @sergii-tkachenko ?